sky/clouds/service_catalog/common.py

```diff
@@ -96,23 +96,37 @@ def _need_update() -> bool:
         if _need_update():
             url = f'{constants.HOSTED_CATALOG_DIR_URL}/{constants.CATALOG_SCHEMA_VERSION}/{filename}'  # pylint: disable=line-too-long
             update_frequency_str = ''
+            filename_str = f' ({filename})'
             if pull_frequency_hours is not None:
                 update_frequency_str = f' (every {pull_frequency_hours} hours)'
-            with backend_utils.safe_console_status(
-                    f'Updating {cloud} catalog{update_frequency_str}'):
+            with backend_utils.safe_console_status((f'Updating {cloud} catalog'
+                                                    f'{filename_str}'
```

Comment:
Do you think `Updating {cloud} catalog: {filename}` will be better, since we already have a `()` for `update_frequency`?

Reply:
That makes sense! Updated.